app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -2376,7 +2376,7 @@ sealed class VaultAddEditEvent {
      */
     data class NavigateToGeneratorModal(
         val generatorMode: GeneratorMode.Modal,
-    ) : VaultAddEditEvent()
+    ) : BackgroundEvent, VaultAddEditEvent()
```

Comment:
Why does this one need to happen in the background?

Reply:
That is a great catch - this was left in by accident after I was testing some things. Thank you!